data/workspaces/boron.inv

```diff
@@ -154,21 +152,32 @@
                     </Property>
                     <Property type="org.inviwo.SimpleLightingProperty" identifier="lighting">
                         <Properties>
-                            <Property type="org.inviwo.FloatVec3Property" identifier="lightPosition">
-                                <semantics semantics="Spherical" />
-                                <minvalue x="0" y="0" z="0" />
-                                <maxvalue x="1" y="1" z="1" />
-                                <value x="-2.1760373e-08" y="0.4978193" z="0.55791628" />
+                            <Property type="org.inviwo.PositionProperty" identifier="lightPosition">
+                                <Properties>
+                                    <Property type="org.inviwo.OptionPropertyEnumInt" identifier="referenceFrame">
+                                        <selectedIdentifier content="view" />
+                                    </Property>
```

Comment:
Deafult?

Reply:
Default state for this one is world space.